modules/installation-complete-user-infra.adoc

```diff
@@ -147,7 +147,7 @@ If the pod logs display, the Kubernetes API server can communicate with the
 cluster machines.
 
 ifndef::ibm-power[]
```

Comment:
@bobfuru I've verified with @alkleinrh currently it still works differently for IBM Z. 
Please add ibm-z to ifndef:` ifndef::ibm-power,ibm-z[]` also to endif in line 158: `endif::ibm-power,ibm-z[]`

ifdef::ibm-z[]
. Additional steps are required to enable multipathing. Do not enable multipathing during installation.
+
See "Enabling multipathing with kernel arguments on {op-system}" in the _Installing on bare metal_ documentation for more information.
endif::ibm-z[]

Reply:
Thank you, @SNiemann15. I believe I have updated this PR based on your feedback. PTAL to verify, thanks!

Reply:
/LGTM